content/en/blog/custom-metrics-autoscaling-new-deploy.md

```diff
@@ -0,0 +1,596 @@
++++
+title = "Autoscaling on Custom Metrics in New Deploy Functions"
+date = "2022-05-11T14:43:46+05:30"
+author = "Ankit Chawla"
+description = "Implementing custom metrics for HPA scaling"
+categories = ["Fission"]
+type = "blog"
++++
+
+
+Autoscaling is one of the key features of kubernetes because of its capability to scale up or down according to the load. This is pretty useful as optimizes cost and no human interference is required for scaling. It is intelligent enough to make its own decisions on scaling matters.
```

Comment:
- Kubernetes K in caps, please verify across the document
- Can replace the second sentence with - _it is useful as it optimizes costs with minimum human intervention_
- Can replace the last line with - _Autoscaling adjusts your applications and resources based on the rise and fall in the demand_